client/package.json

```diff
@@ -9,6 +9,7 @@
     "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "jsonwebtoken": "^8.5.1",
+    "material-ui-icons": "^1.0.0-beta.36",
```

Comment:
We have @material-ui/icons already. What did you need the beta?

Reply:
yeah -- I actually uninstalled it but it didn't make it through the commit. This one fixed a bug I had, which was caused by not importing it w the @ symbol. This one allowed me to do that, but I figured it out and removed it.